carve-lockpicks.lic

```diff
@@ -0,0 +1,114 @@
+=begin
+	Need these settings in your yaml:
+		#Stow locations for Grandmaster's, Master's, and anything less.
+    grand_container: backpack
+    master_container: haversack
+    trash_container: cloak
+    #Whether to initial Grandmaster's with an extra carve.
+    initial: true
+=end
+
+custom_require.call(%w[common common-items common-crafting])
+
+class CarveLockpicks
+
+  def initialize
+    @settings = get_settings
+    @bag = @settings.crafting_container
+    @belt = @settings.engineering_belt
+	  @grandbag = @settings.grand_container
+	  @masterbag = @settings.master_container
+	  @trashbag = @settings.trash_container
+	  @initial = @settings.initial
+    @tool = "carving knife"
```

Comment:
Why use an instance var when this is a constant string?

Reply:
removed.